### PR TITLE
[pix2pix]: ML code

### PR DESCRIPTION
**Pull request by Meadosc**
Add ML code for pix2pix

### CODE REVIEW

File: pix2pix/scripts/get_data.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import tarfile
+import requests
+
+try:
+    from scripts.helper import get_directory
+except ModuleNotFoundError:
+    from helper import get_directory
+
+url = "https://people.eecs.berkeley.edu/~tinghuiz/projects/pix2pix/datasets/facades.tar.gz"
```
Comment (unrahul) [inline pix2pix/scripts/get_data.py]: @cord add to readme this attribution for facades dataset that we used: https://github.com/phillipi/pix2pix/blob/master/datasets/bibtex/facades.tex

Comment (Meadosc) [reply pix2pix/scripts/get_data.py]: Citation is added

File: pix2pix/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Citation
+```
+@article{pix2pix2017,
+  title={Image-to-Image Translation with Conditional Adversarial Networks},
+  author={Isola, Phillip and Zhu, Jun-Yan and Zhou, Tinghui and Efros, Alexei A},
+  journal={CVPR},
+  year={2017}
+}
+```
```
Comment (unrahul) [inline pix2pix/README.md]: Add one more for the dataset from here : http://cmp.felk.cvut.cz/~tylecr1/facade/

File: pix2pix/scripts/get_data.py (hunk 1 above)
Comment (unrahul) [reply pix2pix/scripts/get_data.py]: add  one more for the dataset: http://cmp.felk.cvut.cz/~tylecr1/facade/

File: pix2pix/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+* model (string)
+    * continue - run models named 'generator_model.h5' and 'discriminator_model.h5' in the 'models' directory
+    * checkpoint - run the latest model in the 'checkpoints' directory
+* epochs (int)
+* batch_size (int)
+* cycles (int)
+
+#### Inference
+```bash
+python infer.py
```
Comment (DnPlas) [inline pix2pix/README.md]: Should we mention that if running this you should uncomment the last line of infer.py? Also, we need an img as argument

Comment (unrahul) [reply pix2pix/README.md]: Ah good point. @cord ^^

**Pull request merged**